.pre-commit-config.yaml

```diff
@@ -7,9 +7,7 @@ exclude: |
   # Repos using Sphinx to generate docs don't need prettying
   ^docs/_templates/.*\.html$|
   # You don't usually want a bot to modify your legal texts
-  (LICENSE.*|COPYING.*)|
-  # Exclude __init__.py files
-  __init__\.py$
```

Comment:
Can you skip pre-commit for __init__.py? Otherwise it'll remove all the imports.

Reply:
This will not happen. ruff.toml is configured not to removed unused imports in init files. Btw precommit is already passing and you can see that it has not removed any imports from the init files.

Reply:
Ok great!